sherpa/models/regrid.py

```diff
@@ -565,7 +565,7 @@ def eval_non_integrated(self, pars, modelfunc, data_grid, eval_grid,
             The parameters for the model.
         modelfunc : callable
             The model to evaluate. It is called as
-            modelfunc(pars, x, **kwargs)
+            `modelfunc(pars, x, **kwargs)`
```

Comment:
```suggestion
            ``modelfunc(pars, x, **kwargs)``
```
We now use single backticks for stuff that's linked (like sherpa.utils.akima.akima), but that means to format it as "just text that looks like code" we know need ot use double backticks.

Reply:
Note that this will work (so I'm going to approve), but I hope that at some point in the future we can active sphinx warnings to links that don't resolve to an object to catch typos, moved objects etc. `modelfunc(pars, x, **kwargs)` won't resolve to a Pytohn class in the docs, so this would raise a warning when we do that, but we can also deal with it when we get to it (maybe never).